jquery.validate.unobtrusive.js

```diff
@@ -24,7 +24,7 @@
 
     function escapeAttributeValue(value) {
         // As mentioned on http://api.jquery.com/category/selectors/
-        return value.replace(/([!"#$%&'()*+,./:;<=>?@\[\\\]^`{|}~])/g, "\\\\$1");
+        return value.replace(/([!"#$%&'()*+,./:;<=>?@\[\\\]^`{|}~])/g, "\\$1");
```

Comment:
It was trying to fix #33. The actual problem there was what I commented a little bit ago, that his data-val-for was subtly wrong. My best bet for why we thought it was correct at the time is basically that I was testing the wrong stuff. I think I had hand-wrote his html into my test app and in doing so inadvertently fixed the typo, and my test app was probably also serving the un-updated *.min.js.

Lessons for me: 1. Always repro and fix in the same setup 2. As much as possible, verify that the code I changed is actually getting run.